### DISCUSSION TITLE
Fulltime maintainer and community bounties - APPLY HERE

### DISCUSSION THREAD

**Discussion by outrolled** [serghey-rodin/vesta#2298] [General]
Hi all, as the next step for VestaCP, we are planning to hire a fulltime maintainer/developer to care for Vesta and its roadmap. In addition to a dedicated maintainer, we are planning to launch bounties that reward important features with a monetary ($) contribution for each task that gets merged into the project.
### Contribute
If you are interested in contributing, please watch the **Projects tab** for new work that will be added there in a few weeks.
### To Apply for the role as a Vesta Insider:
**Role:** VestaCP Maintainer and Developer
**Salary:** Varies with experience (guide of around $2500 USD per month)
**Contract type:** Service Provider (contractor)
**Role Description:** Are you interested in taking VestaCP forward? Do you dream in Bash, PHP, and JS? If so, apply now as a VestaCP Github insider. A full Position description will be provided once we post it in LinkedIn and job boards 7 days from now. Candidates that apply here will be given extra consideration.

If you are interested in applying for the role, please submit an application at: [Outroll Careers: VestaCP Maintainer and Developer](https://careers.outroll.com/?role=vestacp-maintainer&utm_source=github_discussions)
Because of our limited Budget, this role is better suited for people in South America, Eastern Europe, and Southeast Asia.

**Comment by Neustradamus** [comment]
Linked to:
- https://github.com/outroll/vesta/issues/2280
- https://github.com/outroll/vesta/discussions/2296
- https://github.com/outroll/vesta/discussions/2298
- https://github.com/outroll/vesta/discussions/2299
- https://github.com/outroll/vesta/discussions/2300
- https://github.com/outroll/vesta/issues/2312
- https://github.com/outroll/vesta/discussions/2315
- https://github.com/outroll/vesta/issues/2321
